packages/demo/src/configuration/blocks/effects/directionalBlurBlock.ts

```diff
@@ -63,18 +63,16 @@ export class DirectionalBlurShaderBinding extends ShaderBinding {
 
     /**
      * Creates a new shader binding instance for the DirectionalBlur block.
-     * @param parentBlock - The parent block
      * @param inputTexture - The input texture
      * @param blurHorizontalWidth - The horizontal blur width
      * @param blurVerticalWidth - The vertical blur width
      */
     constructor(
```

Comment:
I assume DirectionalBlurBlock isn't disableable because of something to do with its role in BlurBlock? 
If so, will there be a similar issue with BlackAndWhiteBlock (which is part of BlackAndWhiteBlurBlock)?

Reply:
**Not necessarily because of its role in BlurBlock, but rather because its a subnode of an AggregateBlock

Reply:
We could make it disableable, but then we'd need to create a disable input connection point on the BlurBlock and wire it up, which we can always do later if we want to.  AggregateBlocks are not automatically disableable, which is no different than before.